bitnami/rabbitmq/templates/networkpolicy.yaml

```diff
@@ -25,6 +25,9 @@ spec:
             matchLabels:
               app: {{ template "rabbitmq.name" . }}
```

Comment:
Please, replace this rule by the one you are providing. No other resource is using those matchLabels. 😃